### PR TITLE
Mc more user info 9

### PR DESCRIPTION
**Pull request by molly-clabots**
#10 

New Categories are Favorite Artist, Show and Venue.

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,6 @@
...
-venv
-
-"Postgres PW : qcdq7Hs218q5ejP5"
-"Sql User Info": user : 'rocker' Password : '3f32f4361r'
-=======
 env
 lmnop.sqlite
-migrations
->>>>>>> master
+venv
```
Comment (mekdeshub123) [inline .gitignore]: remove venv

File: .gitignore [hunk 2]
```diff
@@ -1,12 +1,6 @@
...
+.vscode
```
Comment (mekdeshub123) [inline .gitignore:6]: remove .vscode

**Review by mekdeshub123 — CHANGES_REQUESTED** [review]

**Comment by Di30an** [comment]
Ran this locally and made all the necessary migrations. The website is running but I cannot modify from /user/profile/pk page. Is it like that on your end? I can modify and save in the admin console and save them however these are not showing up on the user page. 

LMNOP_ADMIN_LOGIN

Cannot seem to modify from the users page or show the information that was saved in the admin console. 


LMNOP_WEB

**Comment by molly-clabots** [comment]
Yeah I'm adding that feature for the other branch that I'm working on. I forgot to switch branches when adding that code. I plan on cleaning up the branch. This branch is only for adding the extra user info to the model.

**Comment by Di30an** [comment]
The modeling seems fine. However the last part of  issue #9  'A user will be able to view all of their information. Other users will be able to view public information (so not email, any other private fields).' 

This is what the /user/profile/<pk> looks like on my end. 

image

Ran git fetch before I checked out your branch this morning so it should be current. Are you adding this issue to your other issue and pushing that as one big branch?

**Comment by molly-clabots** [comment]
Yes! I forgot to change branches when I started working on 19

**Comment by Di30an** [comment]
> Yes! I forgot to change branches when I started working on 19

So should we close this issue and merge it with issue 19? You can also check out old commits using git log and finding an old commit.

**Pull request merged**